Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:12 AS base
+WORKDIR /app
+
+
+COPY ./package*.json ./
+RUN npm install && npm cache clean --force
+
+COPY . .
+RUN npm run build
+
+
+
+FROM node:12-alpine
+WORKDIR /app
+COPY --from=base /app/node_modules ./node_modules
```

Comment:
you copied full node_modules with devDependencies. It results in large image with a lot of unused dependencies. i know that result image size is less then 500Mb but still is it better to remove unused files from resulted build